react/src/components/QrReader/components/QrReaderScanner.tsx

```diff
@@ -40,77 +38,87 @@ const isMediaDevicesAPIAvailable = () => {
   return isMediaDevicesAPIAvailable;
 };
 
-export const QrReaderScanner: React.FC<QrReaderScannerProps> = ({
+export function QrReaderScanner({
+  multiScan,
```

Comment:
QrReaderScanner is also used in `PackingScanBoxOrFindByLabelOverlayContent.tsx` so you have to add the multiscan argument there.